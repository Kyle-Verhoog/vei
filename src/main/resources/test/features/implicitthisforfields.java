public class Implicitthisforfields {
  public Implicitthisforfields() {}
  protected int x;
  public int m() {
    return x;
  }
}