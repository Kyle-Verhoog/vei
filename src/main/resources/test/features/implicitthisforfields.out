PUBLIC ┃ public
CLASS ┃ class
IDENTIFIER ┃ A
{ ┃ {
PUBLIC ┃ public
IDENTIFIER ┃ A
( ┃ (
) ┃ )
{ ┃ {
} ┃ }
PROTECTED ┃ protected
INT ┃ int
IDENTIFIER ┃ x
; ┃ ;
PUBLIC ┃ public
INT ┃ int
IDENTIFIER ┃ m
( ┃ (
) ┃ )
{ ┃ {
RETURN ┃ return
IDENTIFIER ┃ x
; ┃ ;
} ┃ }
} ┃ }
